src/Entities/Blacklist.php

```diff
@@ -143,6 +149,24 @@ public function setExpiredTime($value)
         return $this->setAttribute('expiredTime', $value);
```

Comment:
Would be better to call the new methods in this setter and other deprecated setters/getters.